VimeoUpload/Upload/Model/VideoSettings.swift

```diff
@@ -29,18 +29,49 @@ import Foundation
 class VideoSettings: NSObject
 {
     var title: String?
+    {
+        didSet
+        {
+            if let title = self.title
+            {
+                self.title = trim(title)
+            }
```

Comment:
Let's make the argument to trim() optional so we don't have to check `if let title = self.title`